src/explore-education-statistics-common/src/modules/table-tool/components/TimePeriodDataTable.tsx

```diff
@@ -238,6 +284,10 @@ const TimePeriodDataTable = forwardRef<HTMLElement, Props>(
         excludedFilters,
       );
 
+      const showExcludedRowsOrColumnsWarning =
```

Comment:
Same as earlier, 'excluded' might be the wrong term - perhaps `showMissingRowsOrColumnsWarning` would be better?

Reply:
Done